### ISSUE TITLE
Do beta (v3) docs exist?   The 'v3 Documentation' link goes to v2 Docs currently.

### ISSUE THREAD

**Issue by ArthurD** [swisnl/jQuery-contextMenu#759]
The repo's v3 branch README has a link to go to the 'v3 docs' but it actually takes you to the v2 docs.  See screenshot:
<img width="1035" alt="v3-docs-missing" src="https://user-images.githubusercontent.com/109532/144755485-f879db63-1ca4-4150-a6c2-18d2964d542e.png">

I suspect a decent bit of the confusion is related to this -- the v3 code is much better & is overall just far easier to work with (especially the styling) -- but without documentation, it's a lot of fumbling around to try to figure out what the usage API is.  

Personally, I had inadvertently been trying to use v2 syntax with the v3 codebase -- causes me lots of confusion.  Much smoother sailing now that I've realized the issue, though unfortunately I'm going to stick with the v2 implementation, at least _for now_, at least until v3 has some documentation/examples.  And maybe it already does and I'm just too dense to find/see them.

Nonetheless -- awesome project.  Please don't take this as criticism at all BTW!  The project is great and, especially the improvements in v3, it looks fantastic & it absolutely satisfies our requirements & more.  If anything, I just wanted to relay my experience in hopes of helping the project & others in the future, even if in a _very_ small way!

Thank you!  :)

**Comment by bbrala** [comment]
Yeah, not really, i tried back then to upgrade to something new. But didnt end up in a state i was happy about. :(
